### PR TITLE
Sounds for active portable air pumps

### PR DESCRIPTION
**Pull request by Capsandi**
## About The Pull Request

Adds sound to portable pumps when they're operating.
Got the sound from recording my tire pump, the startup and ending sounds were made by pitchbending it
Here's a vid of me walking around it, sound only

https://user-images.githubusercontent.com/38051413/163568313-b01d7ab4-de5b-48dd-b4d8-d4873965a50e.mp4

## Why It's Good For The Game

With current sprites, the only indication a pump is on is a tiny fan and a green pixel. That's a drought of feedback for something which could be flooding the room with co2 or hydrogen.

## Changelog

:cl:
soundadd: Portable air pumps are no longer silent
/:cl:

### CODE REVIEW

File: code/datums/looping_sounds/machinery_sounds.dm [hunk 1]
```diff
@@ -149,3 +149,15 @@
...
 	falloff_distance = 2
 	falloff_exponent = 5
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/datum/looping_sound/pump
+	start_sound = 'sound/machines/pump/pump_start.ogg'
+	start_length = 0.8 SECONDS
+	mid_sounds = 'sound/machines/pump/pump_mid.ogg'
+	mid_length = 1.3 SECONDS
```
Comment (LemonInTheDark) [inline code/datums/looping_sounds/machinery_sounds.dm:159]: If at all possible you want sound snippets to be small. Byond will only update "directional" sound when we play a new audio slice, so if they're too long like this you'll get really obvious "jumping"

**Comment by Serxule** [comment]
sounds pixelated/bitcrushed imo

**Comment by zxaber** [comment]
>industrial_tractor.ogg
It's telecomms all over again

File: code/datums/looping_sounds/machinery_sounds.dm (hunk 1 above)
Comment (Capsandi) [reply code/datums/looping_sounds/machinery_sounds.dm]: I've shortened it with the latest commit and walking around it sounds much smoother now, thanks for the suggestion

File: code/datums/looping_sounds/machinery_sounds.dm [hunk 2]
```diff
@@ -149,3 +149,15 @@
 	volume = 20
 	falloff_distance = 2
 	falloff_exponent = 5
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (Mothblocks) [inline code/datums/looping_sounds/machinery_sounds.dm:153]: remove all of these they're fugly and ruin documentation

Comment (Capsandi) [reply code/datums/looping_sounds/machinery_sounds.dm]: Done

**Comment by ShizCalev** [comment]
that sound is terrible. absolutely not from me. (i dig the idea, just not the sound.)

**Comment by Capsandi** [comment]
Alright I'm going to close this and re-open it in a new pr once i can find/make some crisper sounds

**Pull request closed without merging**